jwql/website/static/mnemonic_exploration_result_table.csv

```diff
@@ -0,0 +1,14 @@
+description,sqlDataType,subsystem,tlmIdentifier,tlmMnemonic,unit
```

Comment:
@Johannes-Sahlmann Is there a reason to commit this data file to the repository?

Reply:
this was an accidental commit. I removed it.